app/app.go

```diff
@@ -187,7 +187,7 @@ var (
 	maccPerms = map[string][]string{
 		authtypes.FeeCollectorName:      nil,
 		distrtypes.ModuleName:           nil,
-		minttypes.ModuleName:            {authtypes.Minter},
+		minttypes.ModuleName:            {authtypes.Minter, authtypes.Burner},
```

Comment:
@riley-stride what is this change?

Reply:
The mint module needs permission to burn tokens because it overallocates then burns excess. Not sure why they built it this way, seems like a strange design. We can come back and revisit the design post launch, but better to punt that that to post launch IMO